provisioning/resources/ui/js/components/Overview.tsx

```diff
@@ -42,12 +42,15 @@ export class Overview extends React.Component<{}, {}> {
         <h3>3. Understanding how Snowplow Mini works</h3>
         <h3>Quicklinks: </h3>
         <ul>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/understanding-your-pipeline/what-is-snowplow-mini'}>What is Snowplow Mini?</a></li>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/open-source-components-and-applications/snowplow-mini/snowplow-mini-0-10-1/usage-guide/'}>Usage guide</a></li>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/open-source-components-and-applications/snowplow-mini/snowplow-mini-0-10-1/control-plane-api/'}>Control Plane API</a></li>
```

Comment:
It points to 0.10.1 docs but I assume we'll have new doc pages dedicated to 0.11.0

Reply:
Fixed.

Reply:
thanks!